projects/go-lib/src/lib/components/go-select/go-select.component.html

```diff
@@ -30,7 +30,7 @@
   [virtualScroll]="virtualScroll"
   (scrollToEnd)="onScrollToEnd()"
   (scroll)="onScroll($event)">
-  <ng-container *ngIf="multiple && showSelectAll && items.length > 0">
+  <ng-container *ngIf="multiple && showSelectAll && items?.length > 0">
```

Comment:
This works but now that we're qualifying that `items` is non-null you don't need to have the `> 0` check. `items?.length` should be enough.

Reply:
Good point, I removed it.